### DISCUSSION TITLE
bump node version to 18

### DISCUSSION THREAD

**Discussion by LudovicSterlin** [directus/directus#16194] [Archive]
[Node](https://nodejs.org) just transitioned onto version 18 as LTS.

Is there any plan to move to this version for directus ?

See [Node: Version 18.12.0 'Hydrogen' (LTS)](https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V18.md#2022-10-25-version-18120-hydrogen-lts-ruyadorno-and-rafaelgss)

**Comment by rijkvanzanten** [comment]
Directus will already run under Node 18 as well 🙂  We'll be updating our Dockerfile shortly 👍🏻
